### PR TITLE
feat(passport-gated): stamps as optional parameter

### PR DESCRIPTION
**Pull request by santteegt**
### Summary

Update `passport-gated` info/warning messages and initial parameter setup to cover the latest updates introduced in https://github.com/snapshot-labs/snapshot-strategies/pull/1309

### CODE REVIEW

**Comment by santteegt** [comment]
This is an open source contribution. I'd appreciate it if you guys add the `hacktoberfest-accepted` label to this PR

**Comment by ChaituVR** [comment]
Why this is closed 🤔

**Comment by samuveth** [comment]
Must have been a miss click

File: src/locales/default.json [hunk 1]
```diff
@@ -741,7 +741,7 @@
     "passport-gated": {
       "label": "Gitcoin Passport gated",
       "description": "Protect your space from spam by requiring users to have a Gitcoin Passport to create a proposal.",
-      "invalidMessage": "You need a Gitcoin Passport with score above {scoreThreshold} and {operator} of the following stamps to vote on this proposal: {stamps}. "
+      "invalidMessage": "You need a Gitcoin Passport with score above {scoreThreshold} and {operator} of the following stamps to create a proposal: {stamps}. "
```
Comment (ChaituVR) [inline src/locales/default.json:744]: I think this still be used inside proposal

Comment (ChaituVR) [reply src/locales/default.json:744]:
```suggestion
      "invalidMessage": "You need a Gitcoin Passport with a score above {scoreThreshold} and {operator} of the following stamps: {stamps}. "
```

**Comment by samuveth** [comment]
Continued here as I couldn't edit this PR https://github.com/snapshot-labs/snapshot/pull/4357

**Pull request closed without merging**